README.md

```diff
@@ -126,7 +138,7 @@ $ npm run start:dev
 
 ## Understanding the local development environment
 
-Tackle2 runs in a Kubernetes compatible environment (Openshift, Kubernetes or minikube) and is usually deployed with Tackle2 Operator (OLM).
+Tackle2 runs in a Kubernetes compatible environment (i.e. Openshift, Kubernetes or minikube) and is usually deployed with Tackle2 Operator (OLM).
```

Comment:
Can probably safely change to Konveyor here as well. 
```suggestion
Konveyor runs in a Kubernetes compatible environment (i.e. Openshift, Kubernetes or minikube) and is usually deployed with Konveyor Operator (OLM).
```

Reply:
There probably could be a whole PR dedicated to changing from Tackle / Tackle2 -> Konveyor so disregard if you like.

Reply:
IMHO the `Tackle` is more descriptive at this point as it used all over the place - starting from repo name (as you mentioned), through various files (like `install-tackle`) to k8s namespaces and pod names.